### DISCUSSION TITLE
When determining twin boundaries (e.g., CSL 3), does/can MTEX consider not just a tolerance about the angle but also about the axis?

### DISCUSSION THREAD

**Discussion by StopkaKris** [mtex-toolbox/mtex#2139] [Ask Anything]
Hello,

This question has to do with another question I posted here a while ago: https://github.com/mtex-toolbox/mtex/discussions/1452

I am interested in determining sigma 3 twin boundaries in FCC materials. To do this, I use a mix of MTEX and [DREAM.3D](https://www.dream3d.io/). DREAM.3D can determine twin boundaries in 2D and 3D digital microstructures and allows the user to set a tolerance for what it should consider a sigma 3/CSL 3 boundary.

In MTEX, the line below will determine the percent of twin boundaries with tolerance of 1 degree.

```CSL_3_GB_percentage = 100 * volume(gB.misorientation,CSL(3,ebsd.CS),1*degree);```

In DREAM.3D, the [Merge Twins](https://www.dream3d.io/nx_reference_manual/html/OrientationAnalysis/MergeTwinsFilter.html) filter allows the user to specify not just the angle tolerance but also the axis tolerance. From the DREAM.3D documentation:

>The user can specify a tolerance on both the axis and the angle that defines the twin relationship (i.e., a tolerance of 1 degree for both tolerances would allow the neighboring Features to be grouped if their misorientation was between 59-61 degrees about an axis within 1 degree of <111>, since the Sigma 3 twin relationship is 60 degrees about <111>).

Does/can MTEX consider not just a tolerance on the angle but also on the axis about which the misorientation is computed?

Best regards,
Kris

Krzysztof Stopka

**Comment by kilir** [comment]
Hi,
the angle threshold in mtex relares to the angle between the measured misorientation and the ideal csl3 misorientation. It is not only axis or only angle.
Cheers,
Rüdiger

**Comment by StopkaKris** [comment]
Hi Rüdiger,

Thank you for your quick reply!

Best regards,
Kris

Krzysztof Stopka
